package.json

```diff
@@ -10,13 +10,5 @@
     "build:min": "cd code && yarn gulp vscode-reh-web-linux-x64-min",
     "rebuild-native-modules": "cd code && npm rebuild"
   },
-  "license": "EPL-2.0",
-  "dependencies": {
-    "node-gyp": "^9.4.1"
-  },
-  "engines": {
```

Comment:
VS Code build has a custom check of the node/yarn versions on [the preinstall phase](https://github.com/che-incubator/che-code/blob/main/code/build/npm/preinstall.js). I believe there's no sense in introducing one more very similar verification. So, I suggest relying on the verification script from upstream.

Reply:
moreover: it broke the che-in-che development: https://github.com/eclipse/che/issues/22686